python/cudf/cudf/core/dataframe.py

```diff
@@ -1133,7 +1131,7 @@ def reindex(
             else:
                 df = DataFrame(None, idx).join(df, how="left", sort=True)
                 # double-argsort to map back from sorted to unsorted positions
-                df = df.take(idx.argsort(True).argsort(True).to_gpu_array())
+                df = df.take(idx.argsort(True).argsort(True).mem)
```

Comment:
What's the output of `idx.argsort(...).argsort(...)`? I'd expect it to be an index or column, either way you need to update the accessor.

Reply:
it returns a column. I pushed a fix. Watching the ci now.